templates/example_chart.yml

```diff
@@ -765,7 +765,6 @@ series:
     color: Orange
     yaxis_id: kWh
     unit: kW
-    transform: return x/1000;
```

Comment:
This doesn't work for me as its already in kw, did you change the units on your system

Reply:
Morning @springfall2008 I'm not aware of changing the units on my system for any entities. 

My "'predbat.pv_power" entity already shows in kW. 

'predbat.pv_power' entity:
![2024-09-01 10_46_19-Settings – Home Assistant and 196 more pages - Personal - Microsoft​ Edge](https://github.com/user-attachments/assets/92143283-4469-4392-ada8-dbd9d673040f)

Graph with "transform: return x/1000" removed:
![2024-09-01 10_49_19-Overview – Home Assistant and 196 more pages - Personal - Microsoft​ Edge](https://github.com/user-attachments/assets/8b54d80a-9282-4679-bb7b-0a0c48ea4e0f)

Graph with "transform: return x/1000" (original from main):
![2024-09-01 10_50_38-Overview – Home Assistant and 196 more pages - Personal - Microsoft​ Edge](https://github.com/user-attachments/assets/a5c96041-3f46-48b8-b9c6-3218cfae195f)